docs/README.md

```diff
@@ -0,0 +1,110 @@
+## Controllers
```

Comment:
In here I think we can have a introduction paragraph explaining that we are creating abstraction resources (sg and clustermesh) to define in one resource what we can't in the crossplane resources (like the refs)

Reply:
+1 @csmartins, do an overview explanation and enumerate the controllers to later go into details of each

Reply:
Nice suggestion! I added a section describing the [CRs](https://github.com/topfreegames/provider-crossplane/pull/54/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R5), started the Controllers [section](https://github.com/topfreegames/provider-crossplane/pull/54/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R13) by listing and describing the controllers